services/ui-src/src/forms/mcpar/cpc/cpc.json

```
@@ -11,8 +15,129 @@
         "id": "cpc-1",
         "type": "text",
         "props": {
-          "label": "Temporary placeholder field",
-          "disabled": true
+          "label": "C1.I.1 Program contract",
+          "hint": "Enter the title and date of the contract between the state and plans participating in the managed care program."
+        }
+      },
+      {
+        "id": "cpc-1-b",
+        "type": "date",
+        "props": {
+          "noLabel": true
+        }
+      },
+      {
+        "id": "cpc-2",
+        "type": "text",
+        "props": {
+          "label": "C1.I.2 Contract URL",
+          "hint": "Provide the hyperlink to the model contract or landing page for executed contracts for the program reported in this program."
+        }
+      },
+      {
+        "id": "cpc-3",
+        "type": "radio",
+        "props": {
+          "label": "C1.I.3 Program type",
+          "hint": "What is the type of MCPs that contract with the state to provide the services covered under the program? Select one.",
+          "choices": [
+            {
+              "name": "cpc-3-o1",
+              "type": "choice",
+              "label": "Managed Care Organization (MCO)"
+            },
+            {
+              "name": "cpc-3-o2",
+              "type": "choice",
+              "label": "Prepaid Inpatient Health Plan (PIHP)"
+            },
+            {
+              "name": "cpc-3-o3",
+              "type": "choice",
+              "label": "Prepaid Ambulatory Health Plan (PAHP)"
+            },
+            {
+              "name": "cpc-3-o4",
+              "type": "choice",
+              "label": "Primary Care Case Management (PCCM) Entity"
+            },
+            {
+              "name": "cpc-3-o5",
+              "type": "choice",
+              "label": "Other, specify",
+              "children": [
+                {
+                  "id": "cpc-3-o5-text",
+                  "type": "textarea"
+                }
+              ]
+            }
+          ]
+        }
+      },
+      {
+        "id": "cpc-4",
```

Change to match the form here and in choices, i.e. `cpc-4-o1` → `cpc-4a-o1`, `cpc-4-o5-text` → `cpc-4a-o5-text`, etc.
```suggestion
        "id": "cpc-4a",
```